src/main/resources/edu/cornell/kfs/krad/config/MaintainableXMLUpgradeRules.xml

```diff
@@ -501,38 +448,78 @@
                 <match>edu.cornell.kfs.module.cg.businessobject.CuAward</match>
                 <replacement>org.kuali.kfs.module.cg.businessobject.Award</replacement>
             </pattern>
-            <!-- Moved the following entries over from the "maint_doc_classname_changes" section. -->
+            <!-- Moved entries over from the "maint_doc_classname_changes" section, and made updates as needed. -->
             <pattern>
                 <match>org.kuali.rice.kns.bo.CampusImpl</match>
-                <replacement>org.kuali.rice.location.impl.campus.CampusBo</replacement>
+                <replacement>org.kuali.kfs.sys.businessobject.Campus</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.location.impl.campus.CampusBo</match>
+                <replacement>org.kuali.kfs.sys.businessobject.Campus</replacement>
             </pattern>
             <pattern>
                 <match>org.kuali.rice.kns.bo.Parameter</match>
-                <replacement>org.kuali.rice.coreservice.impl.parameter.ParameterBo</replacement>
+                <replacement>org.kuali.kfs.coreservice.impl.parameter.Parameter</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.coreservice.impl.parameter.ParameterBo</match>
+                <replacement>org.kuali.kfs.coreservice.impl.parameter.Parameter</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.kfs.coreservice.impl.parameter.ParameterBo</match>
+                <replacement>org.kuali.kfs.coreservice.impl.parameter.Parameter</replacement>
             </pattern>
             <pattern>
                 <match>org.kuali.rice.kew.rule.RuleDelegation</match>
-                <replacement>org.kuali.rice.kew.rule.RuleDelegationBo</replacement>
+                <replacement/>
```

Comment:
Is this replacement correct?

Reply:
KFS does not appear to have its own equivalent of Rice's RuleDelegation or RuleDelegationBo objects. Thus, this rule and the one below will remove them from the maintenance XML if they're somehow present.

The base financials upgrade rules were updated to just remove and RuleDelegation or RuleDelegationBo mappings altogether. If you would prefer, I could remove those mappings from our rules file as well.

Reply:
Thank you for explaining this Chad. What maintenance xml do we expect could contain these 2?

Reply:
Thanks for the input. I'll get those unneeded rules removed.